fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -1062,6 +1064,49 @@ public BrokerFileSystem getGooseFSFileSystem(String path, Map<String, String> pr
         }
     }
 
+    public List<TBrokerFileStatus> listLocatedFiles(String path, boolean onlyFiles,
```

Comment:
What is diff between this and `listPath` method?

Reply:
1. `listPath` method in broker uses `FileSystem.globStatus` to access underlying file system, `FileSystem.globStatus` doesn't expose `recursive` option for only-file file listing. Otherwise, `listLocatedFiles` here uses `FileSystem.listFiles` to access file system, which exposes `recursive` option.
2. Make it consistent with the action in `RemoteFileSystem.listLocatedFiles`